src/Microsoft.AspNetCore.SpaServices.Extensions/SpaOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.SpaServices
+{
+    /// <summary>
+    /// Describes options for hosting a Single Page Application (SPA).
+    /// </summary>
+    public class SpaOptions
+    {
+        internal const string DefaultDefaultPageValue = "index.html";
+
+        /// <summary>
+        /// Gets or sets the URL, relative to <see cref="UrlPrefix"/>,
+        /// of the default page that hosts your SPA user interface.
+        /// The typical value is <c>"index.html"</c>.
```

Comment:
Instead of `typical` maybe just say it's the `default`?

Reply:
Done